fleetshard/pkg/central/cloudprovider/awsclient/rds.go

```diff
@@ -261,16 +265,16 @@ func (r *RDS) waitForInstanceToBeAvailable(ctx context.Context, instanceID strin
 }
 
 // NewRDSClient initializes a new awsclient.RDS
-func NewRDSClient(awsRegion, dbSecurityGroup, dbSubnetGroup string, credentials AWSCredentials) (*RDS, error) {
-	rdsClient, err := newRdsClient(awsRegion, credentials.AccessKeyID, credentials.SecretAccessKey, credentials.SessionToken)
+func NewRDSClient(config *config.Config, auth fleetmanager.Auth) (*RDS, error) {
+	rdsClient, err := newRdsClient(config, auth)
 	if err != nil {
 		return nil, fmt.Errorf("unable to create RDS client, %w", err)
```

Comment:
nit: albeit you didn't write this, looking at our code we seem to have settled on:
```suggestion
		return nil, fmt.Errorf("unable to create RDS client: %w", err)
```

Reply:
I think I wrote that, and it's a typo, I also meant to write `: %w`

Reply:
Fixed.